anago

```diff
@@ -757,6 +763,7 @@ archive_release () {
 
   ((FLAGS_yes)) || common::askyorn -y "Archive this release on GS" || return
 
+  copy_logs_to_workdir
```

Comment:
why add this here? don't we call `common::exit` right after this step, which itself will call `common::cleanexit` and then copy the logs?